app/models/user.rb

```diff
@@ -1,4 +1,5 @@
 class User < ApplicationRecord
+  # has_many :posts
```

Comment:
- [ ] Kindly remove commented code. This was not present during the previous review